docs/SimpleFormIterator.md

```diff
@@ -94,6 +95,17 @@ This prop lets you pass a custom element to replace the default Add button.
 </SimpleFormIterator>
 ```
 
+## `clearButton`
+This prop lets you pass a custom element to replace the default ClearAction button
+
+```jsx
+<SimpleFormIterator clearButton={<Button>Clear</Button>}>
```

Comment:
This example is wrong, as a click on this button will do nothing.

Reply:
Example is removed.